apps/api/src/modules/summary/summary.service.ts

```diff
@@ -55,6 +55,11 @@ export class SummaryService extends CoreService<Summary> {
               this.logger.log(
                 `Saving Uploaded File Details for ${modifiedFilename} at ${fileLocation}`,
               );
+
+              if (userEmail) {
+                summary.createdBy = userEmail;
```

Comment:
Update Modifed By too here. Initial one will be both user, later changes done in backend can be kept as done by system later.

Reply:
updated in https://github.com/OsmosysSoftware/osm-transcript-summarizer/pull/37/commits/ae042a610ff3b1e92f5c0584331d2314df0a57fd